style.css

```diff
@@ -683,6 +684,14 @@ footer {
 }
 
 @media only screen and (min-width: 768px) {
+  .desktop {
+    display: inline-block !important;
+  }
+
+  .mobile {
+    display: none !important;
+  }
+
```

Comment:
- Hey, the same issue is applicable here as pointed out in my previous review.

Reply:
- Hey, the same issue is applicable here as pointed out in my previous review in these [lines of code](https://github.com/NitBravoA92/portfolio-setup-and-mobile-first/blob/cf4b0431c838d994ae8ea09ffdcb65f89c297582/style.css#L74-L80) too.